src/services/groupPaymentService.js

```diff
@@ -0,0 +1,35 @@
+import createResponse from '../utils/createResponse';
+
+import cpmsAuth from '../utils/cpmsAuth';
+import cpmsGroupPayment from '../utils/cpmsGroupPayment';
+import Constants from '../utils/constants';
+
+const cardGroupPayment = async (paymentObject, callback) => {
+	try {
+		const authToken = await cpmsAuth(
+			paymentObject.PenaltyType,
+			Constants.cardHolderPresentAuthBody,
+		);
+		if (authToken === false) {
+			console.log('Error authenticating with cpms');
+			callback(createResponse({ body: 'Error authenticating', statusCode: 400 }));
+		}
+		console.log(authToken);
```

Comment:
We really shouldn't log this out. However, as we're only merging to develop, agreed with @ammarhaiderbjss that we will leave this in for the purpose of debugging while we're still developing. We have a backlog item to sanitise all logs before we merge to `master`